tests/test_plugin.py

```diff
@@ -412,8 +421,8 @@ def test_menu_cache_invalidate_after_menucontent_archive(self):
             language=self.language, version__created_by=self.get_superuser()
         )
         placeholder = factories.PlaceholderFactory(source=page_content)
-        menu_content_version = factories.MenuVersionFactory(content__language=self.language)
-        menu_content = menu_content_version.content
+        menu_content_version = factories.MenuVersionFactory.create_batch(2, content__language=self.language)
```

Comment:
I think this shouldn't create a batch of 2, it's possibly broken the tests because they are testing the one version and cache!